src/skai/model/inference_lib.py

```diff
@@ -302,7 +311,7 @@ def run_tf2_inference_with_csv_output(
       model.
     pipeline_options: Dataflow pipeline options.
   """
-
+  
```

Comment:
nit: Please remove this empty line as well (again, for the lint checker)

Reply:
Hi Jihyeon, I already have a version of the code that addresses all these lint issues. Just haven't pushed to github yet.